app/views/coronavirus_landing_page/_page_header.html.erb

```diff
@@ -95,6 +95,16 @@
           </div>
         <% end %>
       </div>
+
```

Comment:
Are we not doing the following from the trello card?
```
Off - Time not announced:
The next live press conference will be shown here
View past coronavirus press conferences on YouTube
```

Reply:
I think we'll have to do that as an iteration

Reply:
ok